### ISSUE TITLE
web design

### ISSUE THREAD

**Issue by bitedgeco** [bitaverage/bitaverage#26]
This is waiting on the logo

https://github.com/bitaverage/bitaverage/issues/2

design should also be applied to social media profiles

**Comment by bitedgeco** [comment]
Would be nice to have an accent color underline under the nav bar menu item for whatever page you are on.

**Comment by bitedgeco** [comment]
Official accent color is
#A61C40     
rgb(166,28,64)

**Comment by bitedgeco** [comment]
I would like to see how the charts look with a white background.

**Comment by bitedgeco** [comment]
IDK why the social icons in the user requested average result have 40 px of left padding. I can not figure out how to remove it.

**Comment by bitedgeco** [comment]
Improving the design will be a continuous ongoing process.
